ci/windows/Dockerfile

```diff
@@ -4,10 +4,10 @@ FROM ghcr.io/ericoporto/min-ags-dev-env:2.0.16
 # if no temp folder exists by default, create it
 RUN IF exist %TEMP%\nul ( echo %TEMP% ) ELSE ( mkdir %TEMP% )
 
-# Windows 10.0.10240 SDK (but only install the .NET 4.6 SDK)
+# .NET Framework 4.7 Developer Pack
 RUN pushd %TEMP% && \
-  curl -fLO https://download.microsoft.com/download/E/1/F/E1F1E61E-F3C6-4420-A916-FB7C47FBC89E/standalonesdk/sdksetup.exe && \
-  start /wait sdksetup /ceip off /features OptionID.NetFxSoftwareDevelopmentKit /quiet /norestart && \
+  curl -fLO https://download.microsoft.com/download/3/5/1/351f9afb-0505-47dc-a42e-34935707f300/NDP47-DevPack-KB3186612-ENU.exe && \
```

Comment:
Instead of using the `O` flag, you can use `-o` flag to rename the output for something (say `netsetup.exe`), so you could use `curl -fL https://download.microsoft.com/download/3/5/1/351f9afb-0505-47dc-a42e-34935707f300/NDP47-DevPack-KB3186612-ENU.exe -o netsetup.exe` instead.

I mention this because in the line below you are calling `start` without passing the name of the executable you want to start. It should be like

```
start /wait NDP47-DevPack-KB3186612-ENU.exe /install /quiet /norestart && \
```

or if you renamed it

```
start /wait netsetup.exe /install /quiet /norestart
```

Reply:
Whoops, didn't quite get the start command. Fixed it.